lib/domain/usecase/send_image_interactor.dart

```diff
@@ -1,20 +1,17 @@
 
 import 'package:fluffychat/presentation/extensions/room_extension.dart';
 import 'package:matrix/matrix.dart';
-import 'package:photo_manager/photo_manager.dart';
-import 'package:fluffychat/presentation/extensions/asset_entity_extension.dart';
 
 class SendImageInteractor {
   Future<void> execute({
     required Room room,
-    required AssetEntity entity,
+    MatrixFile? matrixFile,
```

Comment:
Why you do this, we expect to move the heavy task to `asynchronize` function. This is a kind of heavy task